src/frontend/Sider/Klage/Komponenter/Behandling/Vurdering/HjemmelVelger.tsx

```diff
@@ -67,16 +40,17 @@ export const HjemmelVelger: React.FC<IHjemmel> = ({ settHjemmel, hjemmelValgt, e
                                     hjemmel: e.target.value,
                                 }) as IVurdering
                         );
-                        settVurderingEndret(true);
                     }}
                     hideLabel
                 >
                     <option value={''}>Velg</option>
-                    {Object.keys(hjemmelValgmuligheter).map((nøkkel, index) => (
-                        <option value={nøkkel} key={index}>
-                            {hjemmelValgmuligheter[nøkkel]}
-                        </option>
-                    ))}
+                    {Object.keys(alleHjemlerTilVisningstekst).map(
+                        (hjemmel: string, index: number) => (
+                            <option value={hjemmel as Hjemmel} key={index}>
+                                {alleHjemlerTilVisningstekst[hjemmel as Hjemmel]}
+                            </option>
+                        )
+                    )}
```

Comment:
Tror ikke du trenger indeksen her, kan bare bruke 
```suggestion
                    {Object.keys(alleHjemlerTilVisningstekst).map(
                        (hjemmel: string) => (
                            <option value={hjemmel as Hjemmel} key={hjemmel}>
                                {alleHjemlerTilVisningstekst[hjemmel as Hjemmel]}
                            </option>
                        )
                    )}
```